packages/filesystem/src/browser/filesystem-frontend-module.ts

```diff
@@ -42,7 +42,7 @@ export default new ContainerModule(bind => {
     bind(FileSystemWatcher).toSelf().inSingletonScope();
     bind(FileShouldOverwrite).toFunction(async function (file: FileStat, stat: FileStat): Promise<boolean> {
         const dialog = new ConfirmDialog({
-            title: `The file '${file.uri}' has been changed on the file system.`,
+            title: `The file '${file.uri.substring(file.uri.lastIndexOf('/') + 1)}' has been changed on the file system.`,
```

Comment:
Thanks! Based on our [coding guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#use-short-name), one should use the `labelProvider.getName()` in order to get a proper representation of a filename across each operating system.